magento/adobe-stock-integration
# #issue-902 Added functionality to delete an image from media gallery

### Description (*)
This PR adds an ability to delete an image from Media Gallery.

### Fixed Issues (if relevant)

1. magento/adobe-stock-integration#902: Delete an image from Media Gallery

### Manual testing scenarios (*)

1. Login to Admin panel
2. Open Media Gallery (i.e. Catalog -> Category -> expand Content -> Select from Gallery button)
3. Click on three dots near to image that you want to remove
4. Click on the "Delete" option
<img width="1919" alt="Screenshot 2020-02-19 at 15 50 11" src="https://user-images.githubusercontent.com/31502344/74841382-49824b00-5331-11ea-8e32-0ee794c000f7.png">
5. Click on "Delete image" button
<img width="1773" alt="Screenshot 2020-02-19 at 15 50 39" src="https://user-images.githubusercontent.com/31502344/74841398-4f782c00-5331-11ea-83a4-44e7707d6ed2.png">

As a result, you should see the success message.
<img width="1639" alt="Screenshot 2020-02-19 at 15 50 54" src="https://user-images.githubusercontent.com/31502344/74841474-6a4aa080-5331-11ea-99e3-0df9a978fec1.png">
